File: 6KYU/Two Sum/solution.cpp
std::pair<std::size_t, std::size_t> two_sum(const std::vector<int>& numbers, int target) {
  size_t len = numbers.size();
  
  for(size_t i = 0; i < len-1; i++) {
    for(size_t j = i+1; j < len; j++) {
      if(numbers[i]+numbers[j] == target) {
        return {i, j};
      }
    }
  }
  
  return {0, 0};
}

File: 6KYU/Two Sum/solution.go
package kata

func TwoSum(numbers []int, target int) [2]int {
	lenxs := len(numbers)

	for i := 0; i < lenxs-1; i++ {
		for j := i + 1; j < lenxs; j++ {
			if numbers[i]+numbers[j] == target {
				return [2]int{i, j}
			}
		}
	}

	return [2]int{}
}
